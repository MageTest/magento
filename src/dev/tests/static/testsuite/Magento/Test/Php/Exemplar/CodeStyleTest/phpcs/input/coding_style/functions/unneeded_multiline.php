<?php
/**
 * Function with parameters.
 * With long description.
 *
 * @param string|null $a
 * @param bool $b
 * @return string
 */
function thereGoesFunc($a,
    $b
) {
    if ($a === null) {
        $a = 'Stranger';
    }
    return 'Hello, ' . $a . '!';
}
