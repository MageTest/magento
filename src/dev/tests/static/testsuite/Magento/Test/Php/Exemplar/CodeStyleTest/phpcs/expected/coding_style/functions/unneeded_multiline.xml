<?xml version="1.0" encoding="UTF-8"?>
<config>
    <incomplete/>
    <error line="10"/>
</config>
